add-on/chrome/modules/LoopRooms.jsm

```diff
@@ -746,9 +746,26 @@ var LoopRoomsInternal = {
       eventEmitter.emit("close");
     });*/
 
-    let win = Services.wm.getMostRecentWindow("navigator:browser");
-    win.LoopUI.loadSidebar(roomToken);
-    MozLoopService.openURL(TOC_URL + roomToken);
+    let urlToLoad =
+      Cc["@mozilla.org/supports-string;1"].createInstance(Ci.nsISupportsString);
+    urlToLoad.data = TOC_URL + roomToken;
+
+    let windowArgs =
+      Cc["@mozilla.org/supports-array;1"].createInstance(Ci.nsISupportsArray);
+    windowArgs.AppendElement(urlToLoad);
+
+    // XXX Do we need to do anything more sophisticated (eg handle private
+    // browsing mode)?  If so, we may want to use utilityOverlay.js:openUILinkIn
+    // (which calls utilityOverlay.js:openLinkIn).  Doing this is rather gross,
+    // as we would need to get the most recently used window and then reach in
+    // and invoke window.openUILinkIn, and it's also conceivably fragile, since
+    // this is unlikely to be a supported way to use that API.
```

Comment:
@Standard8: I assume "set a flag", you mean "set an HTML attribute", since we'll be able to use that in CSS classes.  Or did you have something else in mind?